src/main/java/com/jcabi/xml/ClasspathInput.java

```diff
@@ -121,10 +121,12 @@ public String getEncoding() {
     @Override
     public String getStringData() {
         try {
-            return IOUtils.toString(
+            final String data = IOUtils.toString(
                 this.stream,
                 Charset.forName("UTF-8")
             );
+            this.stream.close();
```

Comment:
@longtimeago ``ClasspathInput`` has a default access-level constructor, so it won't be instantiable from outside the library. I think that makes it ok? We should know our own code when using ``ClasspathInput`` throughout the library.